src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Surfaces/Accordion/BitAccordionDemo.razor.scss

```
@@ -15,3 +16,27 @@
 .custom-desc {
     color: brown;
 }
+
+// Handed to the component as Class/Classes, so it renders on markup of the component's own and needs ::deep
+// to reach past the scope; the anchor is the plain wrapper div each of those examples is written inside.
+::deep {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)